### ISSUE TITLE
Summer of Hacks 2016 - event selection

### ISSUE THREAD

**Issue by spikeheap** [jsoxford/jsoxford.github.com#205]
Following on from last year's pretty amazing (if we do say so ourselves) Summer of Hacks, we need to run it again.

The purpose of this issue is to suggest and vote on events, which will help us put together the schedule for the summer.

#### Adding a suggestion

1. Create one comment per event suggestion.
2. For events we've run before it's fine to just add the name.
3. If it's a new event, add a little detail about it, such as a link, or what's involved. Any requirements or pre-requisites would be good to get out here.
4. Poke people on [Slack](https://digitaloxford.slack.com/messages/jsoxford/) to talk about it!

#### Voting and commenting

__Vote on an issue by adding an emoji response__. Obviously, it would be great if we could discern your intent from the emoji :wink:.

Let's try discussing things in [Slack](https://digitaloxford.slack.com/messages/jsoxford/), but feel free to add pertinent comments in. below. We'll probably do housekeeping to roll them into the suggestions or other issues to keep it clean.

#### Things this issue isn't really about

- Protracted discussions about the relative merits of each event
- Organising dates (unless the event is tied to a specific date, e.g. an international hack day)

Part of #204

**Comment by spikeheap** [comment]
[International NodeSchool Day](http://nodeschool.io/international-day/) is on Saturday the 21st of May this year. We ran this before as #103.

**Comment by AverageMarcus** [comment]
[International Nodebots Day](https://github.com/nodebots/nodebotsday) is 30th July

**Comment by spikeheap** [comment]
The [Real Time Hack](https://github.com/jsoxford/jsoxford.github.com/issues/57) was a pretty fun day.

**Comment by AverageMarcus** [comment]
I'd quite like a hack day based around open data or free APIs. 

Relevant: https://github.com/toddmotto/public-apis/blob/master/README.md

**Comment by AverageMarcus** [comment]
Not sure if suitable for a hack day or not but I've been a bit obsessed with chat bots lately (specifically [Hubot](https://hubot.github.com/)) and would love to see some crazy things that people could come up with. I'm loving the rise of "ChatOps" and think it'd be fun to hack with.

**Comment by danielthepope** [comment]
I'd like to see something based on wearables/general hardware hacking (not necessarily robots). By putting together a hack day we could have a bunch of different materials that would be difficult for individuals/beginners to source.
Basically I already have an idea but I need some gloves, conductive thread and more than one brain to work on it.

**Comment by benfoxall** [comment]
@omgmog suggested a VR hackday

**Comment by spikeheap** [comment]
@rumyra mentioned a car-hacking day which could be really exciting (note this is real full-size motorcars this time). Could be really interesting to do with @gfwilliams.

**Comment by AverageMarcus** [comment]
Code Retreat (See #18) 

This was my first taste of JSOxford and have kept coming back since. Really enjoyed it!

**Comment by benfoxall** [comment]
@piersstorey mentioned a festival hack thing which sounded cool:

> Basically it would be something like multiple groups of varied developers meet in a festival / campsite and hack on their project language for a long weekend

**Comment by gfwilliams** [comment]
It'd be fun to get involved in something - not sure what... I'm obviously a bit more interested in the hardware side of things - I think to hack up something interesting you'd probably want to have a few tools and parts within easy reach, which might make camping more difficult.

Might be able to sort something out with the [Oxford Hack Space](http://oxhack.org/)?

... having said that, you could still do soldering, 3D printing (maybe even laser cutting ;) in a tent...

**Comment by omgmog** [comment]
The VR hack day I suggested recently would be more of an applied workshop type thing (perhaps structured like [Node School](http://jsoxford.com/2014/node-school/) was?). Specifically using Google Cardboard kits (as they're cheap/nominal to supply at scale) and [A-frame](https://aframe.io) for rapidly creating cardboard-ready VR stuff.

Having suggested this though I've not done much in the way of progressing the idea further, and I think my summer is rapidly filling up!

**Comment by peterjwest** [comment]
I am 100% on board with this @omgmog

**Comment by lpmi-13** [comment]
I would personally be very interested in anything related to Education, but my best idea for that would also work with a hardware hackday. I would be happy to bring a bunch of these (http://formcard.co/), since they seem cheap enough, and I need something that can be shaped to the hard palate of the mouth.

TL;DR - another suggestion for hardware hackdays

**Comment by AverageMarcus** [comment]
I'm rubbish at making games but it might be fun to do a game hack day. 

One of the assignments I had at uni was to write the code for a "character" in a game which we were given the API. The lecturer then played all the characters against each other. I think something similar could be fun.

**Comment by spikeheap** [comment]
Great, thanks everyone for input and votes. We're discussing it this week, but the current standings are:

1. Games! `+4`
2. Code Retreat `+4`
3. Wearables / hardware `+3`
4. Open Data `+3`
5. RTHack `+3`

__That's not a final list of events__, but it does give us a good idea of what people are keen for. 

We'll close this issue down and open individual issues for the events shortly. Cheers!
